_posts/2016-06-22-metronome-practice.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Simple Ways to Rule Rhythm"
+author: "Hayley MacLeod"
+categories: practice
+header: /assets/img/2016-07-14/metronome.png
+---
+
+Rhythm is often under-rated both in practice and teaching. A strong sense of rhythm is one of the distinguishing features of a great musician. In a performance, inconsistent rhythms can lead to feelings of unease. What’s the fastest way to rule rhythm? __The metronome__.
+
+## 4 Ways to 'Level Up' Your Rhythm Game
+
+![](/assets/img/2016-07-14/tambourine.jpg){:width="75%"}
+
+#### Finding the Beat
+
+The beat of a song is similar to the tick-tock of a clock. Finding the beat in a song is like listening for the sound of a clock within a room. You may not recognize it until you focus on the sound. Listen to a [song](https://www.youtube.com/watch?v=DohRa9lsx0Q&list=PLK9Sc5q_4K6aNajVLKtkaAB1JGmKyccf2) and find the beat. Clap your hands, snap your fingers or tap your toes.
```

Comment:
Can embed youtube videos in place of pictures. (Linking in this way works too, just giving extra formatting options)